### ISSUE TITLE
Cannot use Get or others with default route when using group

### ISSUE THREAD

**Issue by duchacekjan** [FastEndpoints/FastEndpoints#806]
I when creating group in which I want to have default route i.e. group /api/my-feature and full route is GET /api/my-feature I have to configure Endpoint like this:
```csharp
Get("");
Group<MyFeatureGroup>();
```
instead of
```csharp
Get();
Group<MyFeatureGroup>();
```
because Group requires routes. It seems unnecessary. Could you explain it to me why is this must have or is it just oversight and it will be fixed?

**Comment by dj-nitehawk** [comment]
typically it's done like this:
```cs
sealed class MyGroup : Group
{
    public MyGroup()
    {
        Configure("/api/my-feature", _ => { });
    }
}

sealed class MyEndpoint : Ep.NoReq.NoRes
{
    public override void Configure()
    {
        Get("/"); //better to be explicit here and show that it's the root of the feature/group
        Group<MyGroup>();
    }
}
```
i'm not liking the idea of an empty `Get()` call in endpoint configuration as that looks a bit weird when a route is not specified. from a code reader perspective, it doesn't sit right. like... "what the hell is it getting?" hope you can see my point 😉

**Comment by duchacekjan** [comment]
Thank you for prompt answer and explanation. I will just stick to string.Empty as "/" looks weird on swagger ui

**Comment by duchacekjan** [comment]
Thank you for really quick explanation. However my concern lies with swagger ui now. If I use string.Empty or "/" as you suggested then when using Group the endpoint will be ending with "/" which does look weird.

**Comment by dj-nitehawk** [comment]
that is an oversight...
fixed in `v5.30.0.22-beta`
